src/ompl/control/planners/syclop/Syclop.h

```diff
@@ -330,21 +329,21 @@ namespace ompl
                 }
                 /** \brief The cells of the underlying coverage grid that contain tree motions originating from
                     direct connections along this adjacency */
-                std::set<int> covGridCells;
+                std::set<int> covGridCells = {0};
```

Comment:
I think you want to create an empty set, not a singleton set with 0 as its element.
```suggestion
                std::set<int> covGridCells = {};
```